### PR TITLE
Add checkstyle, add release workflow in Grabl, delete CircleCI file

### PR DESCRIPTION
**Pull request by alexjpwalker**
## What is the goal of this PR?

To allow protocol to build in Grabl

## What are the changes implemented in this PR?

Add checkstyle, add release workflow in Grabl, delete CircleCI file, and ensure that every file has an AGPL license

### CODE REVIEW

File: .bazelrc [hunk 1]
```diff
@@ -1,3 +1,20 @@
...
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
 build --incompatible_strict_action_env
 run --incompatible_strict_action_env
 test --incompatible_strict_action_env
```
Comment (lolski) [inline .bazelrc:20]: The bazelrc doesn't have the buildbuddy stuff here. is it expected?

Compare it with the bazelrc of grakn-core for example: https://github.com/graknlabs/grakn-2.0/blob/master/.bazelrc

**Review by lolski — COMMENTED** [review]
Looks mostly good, but I've got one question.

File: .bazelrc (hunk 1 above)
Comment (alexjpwalker) [reply .bazelrc:20]: What's missing? It looks good to me

Comment (lolski) [reply .bazelrc:20]: ah lol yeah it's actually there when i expanded it. was just delusional

**Review by lolski — APPROVED** [review]

**Pull request merged**